src/scripts/submit.ts

```diff
@@ -0,0 +1,49 @@
+import { Network, PoolClient, Request, TxOptions } from '@blend-capital/blend-sdk';
+import { config } from '../utils/env_config.js';
+import { AddressBook } from '../utils/address_book.js';
+import { logInvocation, signWithKeypair } from '../utils/tx.js';
+
+async function mock(addressBook: AddressBook) {
```

Comment:
Is this for debugging?

Reply:
I've found some use of having a file that just executes a function on the pool so I left it, but yes